tanzu-java-web-app/README-native.md

```diff
@@ -0,0 +1,17 @@
+
+## Deploying the sample on TAP built as a GraalVM native image
+
+In order to build this sample as a GraalVM native image, you can use the following command:
+
+```sh
+tanzu apps workload create tanzu-java-web-app \
+  --file ./config/workload.yaml \
+  --local-path . \
+  --source-image "${SOURCE_IMAGE}" \
+  --build-env "BP_NATIVE_IMAGE=true" \
+  --build-env "BP_MAVEN_BUILD_ARGUMENTS=-Pnative -Dmaven.test.skip=true --no-transfer-progress package -Dspring-boot.aot.jvmArguments='-Dmanagement.health.probes.enabled='true' -Dmanagement.endpoint.health.probes.add-additional-paths='true' -Dmanagement.server.port=8081 -Dserver.port=8080' " \
```

Comment:
This will work, but I'm curious if you have tried using the BP_MAVEN_ADDITIONAL_BUILD_ARGUMENTS build-env instead?  I think you can removed the "-Dmaven.test.skip=true --no-transfer-progress package" arguments.

Reply:
Not sure if the buildpack version that supports this is in TAP yet

Reply:
You still have to add the -Pnative to the BP_MAVEN_ADDITIONAL_BUILD_ARGUMENTS (I believe), but I thought this parameter was available in the current version of the build pack.

Reply:
Anyway, it's not a big deal; you're solution above will still work.